examples/strings/strings.bal

```diff
@@ -35,7 +35,7 @@ public function main() {
     io:println("Concat: ", s4);
 
     // Join strings with a separator.
-    string s5 = ",".'join(hello, ballerina);
+    string s5 = ",". 'join(hello, ballerina);
```

Comment:
We don't need the space right?

Reply:
Thanks. Created the issue https://github.com/ballerina-platform/ballerina-lang/issues/21002 to track this.